generic/htx_test.py

```diff
@@ -151,9 +181,14 @@ def stop_htx(self):
         cmd = 'htxcmdline -shutdown -mdt %s' % self.mdt_file
         process.system(cmd, timeout=120, ignore_status=True)
 
-        daemon_state = process.system_output('/etc/init.d/htx.d status')
-        if daemon_state.decode().split(" ")[-1] == 'running':
-            process.system('/usr/lpp/htx/etc/scripts/htxd_shutdown')
+        if self.run_type == 'rpm':
+            process.system(
+                '/usr/lpp/htx/etc/scripts/htxd_shutdown', ignore_status=True)
+            process.system('umount /htx_pmem*', ignore_status=True)
```

Comment:
maybe adding  logic to uninstall rpm will be good

Reply:
Isn't it similar to installing any package for the test ? meaning it shouldn't harm anything else. Anyway if it is certainly needed we can add, please let me know.

Reply:
ok I just saw we are killing service so thought

Reply:
Ok, just to make sure test should nothing should be running at the end of test.